ggshield/cmd/hmsl/check_common.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Optional
+
+import click
+from requests import HTTPError
+
+from ggshield.core.config import Config
+from ggshield.core.errors import UnexpectedError
+from ggshield.core.text_utils import display_info, pluralize
+from ggshield.verticals.hmsl import Secret, get_client
+from ggshield.verticals.hmsl.collection import PreparedSecrets
+from ggshield.verticals.hmsl.output import show_results
+
+
+def common_check(
+    ctx: click.Context,
+    prepared_data: PreparedSecrets,
```

Comment:
naming: if this argument is an instance of `PreparedSecrets`, then it can be named `prepared_secrets`.

Reply:
Done